### ISSUE TITLE
"engine" tag doesn't get removed

### ISSUE THREAD

**Issue by witsec** [witsec/mobirise-white-label#3]
Right now the extension doesn't properly filter the engine-tag:

```
<section id="top-1" class="engine">
  <a href="https://mobirise.ws">Mobirise Website Builder</a>
  " v4.8.10"
</section>
```

The regular expression needs some adjustments.

**Comment by witsec** [comment]
Could potentially solve this:
`/<section class="engine">.+<\/section>/img`

With this:
`/<section.*class=['"]?engine['"]?[\s\S]*?<\/section>/img`

**Comment by witsec** [comment]
Fixed in v0.5
